qiime/core/type/signature.py

```diff
@@ -133,12 +146,12 @@ def _parse_signature(self, callable, inputs, parameters, outputs):
                                     " parameters in callable signature.")
                 annotated_inputs[name] = ParameterSpec(
                     qiime_type=inputs.pop(name), view_type=view_type,
-                    default=default)
+                    default=default, help_text=help_text.get(name))
```

Comment:
This is the case in a couple places. You might be able to set a `help_text` variable where the `default` and `view_type` are set, but you'll need another name for the `help_text` dict.